docs/locale/ru/LC_MESSAGES/about/credits.po

```diff
@@ -62,7 +62,7 @@ msgstr ""
 "<http://www.opendataservices.coop>`__), при этом было получено содействие "
 "рабочей :doc:`группы по стандарту данных <governance>`."
 
-# 51628ca7072b4f529336319cecfae3dc
+# e739e64e46cb49629b1cb7e80cfbc2dc
```

Comment:
l78 is presented as 

Версию 0.2 написали Джек Лорд, Саймон Уайтхаус и Кади Армстронг ([`Open Data Services Co-operative http://www.opendataservices.coop>`_](https://standard.openownership.org/ru/0.2.0-transations-sync/about/credits.html#id7)),

in my browser (Vivaldi on Ubuntu), ie it shows the url rather than a hyperlink.

Reply:
I pushed a fix for this but it seems to have lost the RU translation altogether :roll_eyes: Investigating..